src/backend/renderer/dummy.rs

```diff
@@ -1,6 +1,9 @@
+//! DummyRenderer
```

Comment:
DummyRenderer actually isn't a very good name since it does do something (tries to access every pixel in an imported shm buffers). Maybe TestRenderer would make more sense?